examples/python/Quantinuum_circuit_submissions.py

```diff
@@ -167,6 +167,7 @@
 
 # Jobs that have been submitted can also be cancelled if needed.
 
+handle = backend.process_circuit(compiled_circuit, n_shots=n_shots)
```

Comment:
Otherwise the next line will raise an error saying the job has already been completed.

Reply:
ok, I commented out that line, this is where using this automated thing on the script instead of the notebook itself gets a little goofy. there are places in the original notebooks that have comments in the code cells, but they get converted to markdown because of `p2j` detecting them as such since the lines starts with a #. And in this case, the best thing is to show the user what the function is in a code cell. I can see how it causes issue here, ... but that's just my thought on using a script to check that the notebooks are up to date... not sure how to get around that